README.md

```diff
@@ -1,2 +1,45 @@
-WIP: This will be a template README.md that shows the basic structure
-that README.md files in LibertyDSNP projects should generally have.
+This template outlines the basic structure that README.md files across Project 
+Liberty should generally have. Not all components will be relevant for all work.
+Think what questions your audience might have and try to answer them in the 
+README.md file. 
+
+## Overview 
+A brief description of the work and the intended audience. 
+
+## Installation 
+A concise but thorough, step-by-step installation guide, written for a someone 
+new to the work.
+
+## Dependencies/Requirements
+List any external dependencies or requirements for running the work. For example, 
+updates to the DSNP.org web site requires the use of the Ghost CMS, which is not 
+managed by Project Liberty. 
+
+## Configuration
+List any useful configuration tips. 
+
+## Examples
+Share examples of how this work could be put into practice. Code examples and 
+screenshots are helpful.
+
+## Roadmap
+If available, highlight an at-a-glance roadmap for future development. 
+
+## Support
+Direct the audience to the [Project Liberty Forums](https://forums.projectliberty.io/)
+or to file a GitHub Issue. If available and it's part of your workflow, include 
+a project-specific email. 
+
+## Contributing
+Note that contributions are welcomed and if there are any areas of particular need. 
+Also link to Project Liberty's overall [contribution guidelines](meta/CONTRIBUTING.md).
+
+## Additional Resources
+Include links to helpful resources, further reading, etc.  
+
+## Acknowledgements
```

Comment:
I'm not sure we should include this section as a matter of course; it's complicated (acknowledgements sections tend to get out-of-date very quickly; also they're inflationary).